lib/client/websocket-tracker.js

```diff
@@ -130,6 +131,8 @@ WebSocketTracker.prototype.destroy = function (cb) {
   }
 
   self.socket = null
+  self.reconnecting = false
```

Comment:
This isn't correct. This boolean needs to stay `true` until a 'connect' event is fired. Setting it to `false` here defeats the point of it.